deploy/charts/approver-policy/values.linter.exceptions

```diff
@@ -1 +1,7 @@
-value missing from values.yaml: nameOverride
\ No newline at end of file
+value missing from values.yaml: nameOverride
+
+# Some false postives
+# See https://github.com/cert-manager/helm-tool/issues/27
+value missing from templates: tolerations
+value missing from templates: affinity
+value missing from templates: nodeSelector
```

Comment:
The `helm-tool lint` command did not have problems with `hostNetwork` or `dnsPolicy` for some reason. Perhaps because those variables are not used in `with ...` statements in the templates.